### DISCUSSION TITLE
top, bottom, left & right properties

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#582] [Q&A]
```
-- ftd.text: FifthTry
sticky: true
top.px: 50
```

top works with sticky but, the parent div remains in its original position only the child div moves from the top, as shown in the attachment. 

![Screenshot from 2023-02-08 15-05-57](https://user-images.githubusercontent.com/119656213/217491803-48c9a92f-429a-4a97-9b03-e16618296b21.png)

can you give an example for these properties.

**Comment by Arpita-Jaiswal** [comment]
These properties can be used with `anchor` and `sticky`.

For anchor: parent
```ftd
-- ftd.column:
width.fixed.px: 400
height.fixed.px: 400
background.solid: pink

-- ftd.text: Hello Parent
anchor: parent
left.rem: 8.3
top.rem: 8

-- end: ftd.column
```


For anchor: window
```ftd
-- ftd.text: Hello Window
anchor: window
left.px: 20
top.px: 30
```


For sticky:
```ftd
-- ftd.text: Sticky Hello world
sticky: true
top.px: 50
left.px: 100
```

**Reply by amitu** [reply]
Let's update this answer when we have created a demo on ftd.dev in appropriate documentation page and add a link so people can easily see the rendered version.
